generators/server/templates/src/test/java/package/web/rest/AccountResourceIT.java.ejs

```diff
@@ -55,89 +54,73 @@ import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
 
+import static com.tngtech.sstestsupport.web.rest.AccountResourceIT.TEST_USER_LOGIN;
```

Comment:
Should this line be removed?

Reply:
If you want to use the constant within the annotation I think you need it, don't you?

Reply:
Ok